spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/RoutingKeyGenerator.scala

```diff
@@ -7,7 +7,7 @@ import com.datastax.spark.connector.cql.TableDef
 
 /** This class computes the routing key of a bound statement. */
 class RoutingKeyGenerator(table: TableDef, columnNames: Seq[String])
-  extends ((BoundStatement) => ByteBuffer) {
+  extends ((BoundStatement) ⇒ ByteBuffer) {
```

Comment:
I think this is something we must agree upon. So far, my IntelliJ does not do that, and we have nice arrows in half of the places and ugly arrows in another half.